website/pages/en/cookbook/grafting.mdx

```diff
@@ -24,6 +24,22 @@ For more information, you can check:
 
 In this tutorial, we will be covering a basic usecase. We will replace an existing contract with an identical contract (with a new address, but the same code). Then, graft the existing subgraph onto the "base" subgraph that tracks the new contract.
 
+## Important Note on Grafting When Upgrading to the Network
+
+> Caution: If you are migrating your subgraph from Subgraph Studio or the hosted service to the decentralized network, it is strongly recommended to avoid using grafting during the upgrade process.
+
+### Why is this Important?
```

Comment:
Are "is" and "this" not capitalized in title case?

Reply:
No - that's a typo - thank you!